bcda/api/v2/api.go

```diff
@@ -15,6 +15,14 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+var h *api.Handler
+
+func init() {
+	// TODO (BCDA-3582) - Since v2 APIs are not available (as of 2020-10-21), we're still
+	// routing requests to the v1 BFD endpoints
+	h = api.NewHandler([]string{"Patient", "Coverage"}, "/v1/fhir")
```

Comment:
Hard coding the supported resource types (instead of env var). We'll want to do some verification on BCDA end before enabling a new resource type.